### PR TITLE
Remove confusing logging from container output and make webserver URLs prominent

### PR DESCRIPTION
**Pull request by jdbaldry**
Adds requirement for either `curl` or `wget` but that should be almost universal. The user receives an error if neither is installed.

It also adds an error message if the user has neither of `docker` or `podman` installed.

Please can you ensure that you are able to run each of the following, on this branch, without error or confusion:
- `make docs`
- `make docs PROJECTS=grafana`

If you feel that any of the other messages are not useful by default, let me know and I can remove them also.

Updated output:

```console
Start building sites … 
hugo v0.115.2 linux/amd64 BuildDate=unknown


                   |  EN  |  DE  |  FR  |  ES  | PT-BR | ZH-CN  
-------------------+------+------+------+------+-------+--------
  Pages            |  115 |   12 |   12 |   12 |    12 |    12  
  Paginator pages  |    0 |    0 |    0 |    0 |     0 |     0  
  Non-page files   |   13 |    0 |    0 |    0 |     0 |     0  
  Static files     | 1227 | 1227 | 1227 | 1227 |  1227 |  1227  
  Processed images |    0 |    0 |    0 |    0 |     0 |     0  
  Aliases          |   64 |    0 |    0 |    0 |     0 |     0  
  Sitemaps         |    2 |    1 |    1 |    1 |     1 |     1  
  Cleaned          |    0 |    0 |    0 |    0 |     0 |     0  

Built in 1372 ms
Watching for changes in /hugo/{content,data,i18n,layouts,package.json,static}
Use watcher with poll interval 700ms
Watching for config changes in /hugo/config/_default, /hugo/config/docs
Environment: "docs"
Serving pages from disk
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender

View documentation at following URLs:
  http://localhost:3002/docs/writers-toolkit/

Press Ctrl+C to stop the server
```

Old output:

```console
DEBG: Mounting '/Users/jdb/ext/grafana/writers-toolkit/docs/sources' at container path '/hugo/content/docs/writers-toolkit'

Documentation will be served at the following URLs:
  http://localhost:3002/docs/writers-toolkit/

fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Makefile:258: warning: overriding recipe for target 'docs'
docs.mk:91: warning: ignoring old recipe for target 'docs'
/usr/bin/make -j 2 proxy hserver-docs HUGO_PORT=3003
make[1]: Entering directory '/hugo'
fatal: not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Makefile:258: warning: overriding recipe for target 'docs'
docs.mk:91: warning: ignoring old recipe for target 'docs'
website-proxy
rm -rf dist*
hugo server --environment=docs -p 3003 --buildDrafts=false --buildFuture=false --bind 0.0.0.0 --noHTTPCache --minify --poll 700ms --renderToDisk -d dist | grep -Ev 'REF_NOT_FOUND: Ref "/(go|blog)/'
2023/08/10 11:30:13 website-proxy listening on :3002...
Start building sites … 
hugo v0.115.2 linux/amd64 BuildDate=unknown


                   |  EN  |  DE  |  FR  |  ES  | PT-BR | ZH-CN  
-------------------+------+------+------+------+-------+--------
  Pages            |  115 |   12 |   12 |   12 |    12 |    12  
  Paginator pages  |    0 |    0 |    0 |    0 |     0 |     0  
  Non-page files   |   13 |    0 |    0 |    0 |     0 |     0  
  Static files     | 1227 | 1227 | 1227 | 1227 |  1227 |  1227  
  Processed images |    0 |    0 |    0 |    0 |     0 |     0  
  Aliases          |   64 |    0 |    0 |    0 |     0 |     0  
  Sitemaps         |    2 |    1 |    1 |    1 |     1 |     1  
  Cleaned          |    0 |    0 |    0 |    0 |     0 |     0  

Built in 1421 ms
Watching for changes in /hugo/{content,data,i18n,layouts,package.json,static}
Use watcher with poll interval 700ms
Watching for config changes in /hugo/config/_default, /hugo/config/docs
Environment: "docs"
Serving pages from disk
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at http://localhost:3003/ (bind address 0.0.0.0)
Press Ctrl+C to stop
```

### CODE REVIEW

File: docs/make-docs [hunk 1]
```diff
@@ -419,6 +435,50 @@ POSIX_HERESTRING
...
 
+await_build() {
+  url="$1"
+  req="$(if command -v curl >/dev/null 2>&1; then echo 'curl -s -o /dev/null'; else echo 'wget -q'; fi)"
+
+  sleep 2
+
+  if ${req} "${url}"; then
+    echo
+    echo "View documentation at following URLs:"
```
Comment (osg-grafana) [inline docs/make-docs:446]:
```suggestion
    echo "View documentation locally:"
```
Yay, my feature request is getting worked on! :) 🎉

**Comment by heitortsergent** [comment]
I think the output looks easier to read 💯 . Question, since some messages are not being displayed by default now, is there a chance there could be something wrong and we wouldn't see it in the output? Or how I would run `make docs` with any flags so I could get the full output?

**Comment by jdbaldry** [comment]
> I think the output looks easier to read 💯 . Question, since some messages are not being displayed by default now, is there a chance there could be something wrong and we wouldn't see it in the output? Or how I would run `make docs` with any flags so I could get the full output?

Thanks for the thoughtful review! There is a way to disable the output filtering and enable debug logging, using the `DEBUG`  environment variable that I have now documented in [9012362](https://github.com/grafana/writers-toolkit/pull/294/commits/901236201ea3ac92c5cbe0ae0fe21089f70a855d).

**Comment by heitortsergent** [comment]
Nice, thanks for adding it to the docs @jdbaldry. 🙇

**Pull request merged**